### DISCUSSION TITLE
Raspberry PI Error

### DISCUSSION THREAD

**Discussion by laststrawman** [stacks-network/stacks-core#3202] [General]
@wileyj I am getting the following error on restart after an Umbrel update ...
ERROR: manifest for wileyj/stacks-blockchain:2.05.0.2.0 not found: manifest unknown: manifest unknown

**Comment by wileyj** [comment]
`docker pull wileyj/stacks-blockchain:2.05.0.2.2`

Note that this is built using the `develop` branch, but in my testing it's worked fine. this is part of the PR https://github.com/stacks-network/stacks-blockchain/pull/3199

I wouldn't count on my repo being maintained though, so until this PR is merged it's definitely better for you to maintain a build of your own (my repo may change randomly as i test things). 

the process is *very* simple on your umbrel over ssh: 
```
$ git clone https://github.com/stacks-network/stacks-blockchain $HOME/stacks-blockchain
$ cd $HOME/stacks-blockchain
$ docker build -f Dockerfile -t stacks-blockchain:latest.
```

Once the build is done, you'll have a local image named `stacks-blockchain:latest` you can use (if you have a docker hub account, you can change the tag to `<username>/stacks-blockchain:latest` and push it there.

**Reply by laststrawman** [reply]
Ok ... I got it built. There was no manage.sh (I guess that was yours?) I copied that over from the previous deploy and it required sample.env so I copied that over as well and now I am getting mainnet not supported.

**Reply by laststrawman** [reply]
Ok ... I got a little further along. I had to copy conf/ and compose-files/ ... now I am running into the same thing I was running into with your branch. "ERROR: manifest for wileyj/stacks-blockchain:2.05.0.2.0 not found: manifest unknown: manifest unknown" I am thinking I can change the common.yaml entry to something like a default I could get it running. I know you said push it to Github as my own branch but not sure how to do that.

**Comment by wileyj** [comment]
two different repos. keep the docker compose one, you need to build an
image for the blockchain
